Source/WebCore/inspector/agents/InspectorCSSAgent.h

```diff
@@ -171,7 +170,7 @@ class InspectorCSSAgent final : public InspectorAgentBase , public Inspector::CS
     HashMap<Node*, Ref<InspectorStyleSheetForInlineStyle>> m_nodeToInspectorStyleSheet; // bogus "stylesheets" with elements' inline styles
     DocumentToViaInspectorStyleSheet m_documentToInspectorStyleSheet;
     HashMap<Document*, HashSet<CSSStyleSheet*>> m_documentToKnownCSSStyleSheets;
-    NodeIdToForcedPseudoState m_nodeIdToForcedPseudoState;
+    HashMap<Inspector::Protocol::DOM::NodeId, HashSet<CSSSelector::PseudoClassType, IntHash<CSSSelector::PseudoClassType>, WTF::StrongEnumHashTraits<CSSSelector::PseudoClassType>>> m_nodeIdToForcedPseudoState;
```

Comment:
Not a big deal, but might be nice to typedef HashSet<...> part of this. It would make this easier to read and reduce the duplication where this same type is used in the implementation file.

Reply:
yeah ideally we'd already have `EnumTraits<CSSSelector::PseudoClassType>` but i think the fact that it's just an `enum` (as opposed to an `enum class`) means that we can't really reason about it as strictly as other things